src/main/java/net/neoforged/neoforge/common/extensions/IFriendlyByteBufExtension.java

```diff
@@ -168,11 +167,16 @@ default <K, V> Map<K, V> readMap(FriendlyByteBuf.Reader<K> keyReader, BiFunction
      * Variant of {@link FriendlyByteBuf#writeMap(Map, FriendlyByteBuf.Writer, FriendlyByteBuf.Writer)} that allows writing values
      * that depend on the key.
      */
-    default <K, V> void writeMap(Map<K, V> map, FriendlyByteBuf.Writer<K> keyWriter, TriConsumer<FriendlyByteBuf, K, V> valueWriter) {
+    default <K, V> void writeMap(Map<K, V> map, FriendlyByteBuf.Writer<K> keyWriter, MapValueWriter<K, V> valueWriter) {
         self().writeVarInt(map.size());
         map.forEach((key, value) -> {
             keyWriter.accept(self(), key);
-            valueWriter.accept(self(), key, value);
+            valueWriter.write(self(), key, value);
         });
     }
+
+    @FunctionalInterface
+    interface MapValueWriter<K, V> {
```

Comment:
I prefix

Reply:
Our functional interfaces generally don't use the `I` prefix.

Reply:
Many do, and they all should.